src/String/Normalize.elm

```diff
@@ -40,12 +41,20 @@ removeDiacritics : String -> String
 removeDiacritics str =
     let
         replace c result =
-            case Dict.get c Diacritics.lookupTable of
-                Just candidate ->
-                    result ++ candidate
+            if Char.toCode c < Diacritics.minCode then
+                result ++ String.fromChar c
 
-                Nothing ->
-                    result ++ String.fromChar c
+            else
+                case
+                    Array.get
```

Comment:
Array lookup is a lot cheaper than Dict lookup. This takes advantage of Char being easily expressed as an array index.

Reply:
I naively thought JS would optimize that.